### PR TITLE
Integration of adaflo reinitialization operation to MeltPoolDG

### PR DESCRIPTION
**Pull request by mschreter**

### CODE REVIEW

File: include/meltpooldg/curvature/curvature_operation_adaflo_wrapper.hpp [hunk 1]
```diff
@@ -48,6 +48,7 @@ namespace MeltPoolDG
                                const Parameters<double> &data_in)
         : scratch_data(scratch_data)
       {
+        (void)normal_vec_dof_idx;
```
Comment (peterrum) [inline include/meltpooldg/curvature/curvature_operation_adaflo_wrapper.hpp:51]: Thx :+1:

**Review by peterrum — APPROVED** [review]
Fine to merge!

**Comment by mschreter** [comment]
Sorry for the misleading title of this PR :)

File: include/meltpooldg/reinitialization/reinitialization_operation_adaflo_wrapper.hpp [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+        normal_vector_operation_adaflo =
+          std::make_shared<NormalVector::NormalVectorOperationAdaflo<dim>>(scratch_data,
+                                                                           reinit_dof_idx,
+                                                                           normal_dof_idx,
+                                                                           reinit_quad_idx,
+                                                                           level_set,
+                                                                           base_in->parameters);
+
+        compute_normal = [&](bool do_compute_normal) {
+          if (do_compute_normal && force_compute_normal)
```
Comment (peterrum) [inline include/meltpooldg/reinitialization/reinitialization_operation_adaflo_wrapper.hpp:78]: The meaning of the bool is different. It means fast or slow computation. So it should not be used in the if-statment if I am not mistaken. 

BTW: why is the the bool not const? Have I forgotten to make it const in adaflo? Nice approach with `force_compute_normal` ;)

**Review by peterrum — APPROVED** [review]
Looks good to me.

**Pull request merged**